src/Calendar/Calendar.tsx

```diff
@@ -167,23 +167,43 @@ const Calendar: RsRefForwardingComponent<'div', CalendarProps> = React.forwardRe
       })
     );
     const timeDropdownProps = pick(rest, DateUtils.calendarOnlyProps);
-    const contextValue = {
-      date: calendarDate,
-      dateRange,
-      disabledDate: isDisabledDate,
-      format,
-      hoverRangeValue,
-      inSameMonth: inSameMonth ?? inSameThisMonthDate,
-      isoWeek,
-      locale,
-      onChangePageDate,
-      onChangePageTime,
-      onMouseMove,
-      onSelect,
-      renderCell,
-      showWeekNumbers,
-      inline
-    };
+    const contextValue = useMemo(
+      () => ({
+        date: calendarDate,
+        dateRange,
+        disabledDate: isDisabledDate,
+        format,
+        hoverRangeValue,
+        inSameMonth: inSameMonth ?? inSameThisMonthDate,
+        isoWeek,
+        locale,
+        onChangePageDate,
+        onChangePageTime,
+        onMouseMove,
+        onSelect,
+        renderCell,
+        showWeekNumbers,
+        inline
+      }),
+      [
+        calendarDate,
+        dateRange,
+        format,
+        hoverRangeValue,
+        inSameMonth,
+        inSameThisMonthDate,
+        inline,
+        isDisabledDate,
```

Comment:
The 'isDisabledDate' function makes the dependencies of useMemo Hook (at line 188) change on every render. To fix this, wrap the definition of 'isDisabledDate' in its own useCallback() Hook